listenbrainz/webserver/static/js/src/SpotifyPlayer.tsx

```diff
@@ -212,7 +212,7 @@ export default class SpotifyPlayer
         }
       })
       .catch((error) => {
-        handleError(error);
+        handleError(error.message);
```

Comment:
Looking at the implementation of handleError, this shouldn't be needed:
https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/webserver/static/js/src/BrainzPlayer.tsx#L164-L178

Reply:
Will fix.